en/sensor/ulanding_radar.md

```diff
@@ -5,8 +5,8 @@ This supersedes the *Aerotenna uLanding Radar* (discontinued) but uses the same
 :::
 
 The *Ainstein* [US-D1 Standard Radar Altimeter](https://ainstein.ai/drone-makers-drone-service-providers/us-d1/) is a compact microwave rangefinder that has been optimised for use on UAVs.
-With a sensing range of about 40 metres, it is useful for applications including terrain following, precision hovering (e.g. for photography), anti-collision sensing etc.
-Particular advantages of this product are that it can operate effectively in all weather conditions and over all terrain types (including water).
+With a sensing range of about 50 metres, it is useful for applications including terrain following, precision hovering (e.g. for photography), anti-collision sensing etc.
+Particular advantages of this product are that it can operate effectively in all weather conditions and over all terrain types (including water). The user manual can be found [here](https://ainstein.ai/wp-content/uploads/2022/04/US-D1-Technical-User-Manual.pdf)
```

Comment:
```suggestion
Particular advantages of this product are that it can operate effectively in all weather conditions and over all terrain types (including water).
The user manual can be found [here](https://ainstein.ai/wp-content/uploads/2022/04/US-D1-Technical-User-Manual.pdf).
```